include/GL/glui.h

```diff
@@ -1587,40 +1654,39 @@ class GLUIAPI GLUI_EditText : public GLUI_Control
     int                 text_x_offset;
     int                 substring_start; /*substring that gets displayed in box*/
     int                 substring_end;  
-    int                 sel_start, sel_end;  /* current selection */
-    int                 num_periods;
```

Comment:
No sh*t. Study GLUI. You don't know how it works.

For one, it should just be a bool. 1 decimal point or 0. And second it's not used anywhere. Use find(',') to see if the number has a period. Don't track the number of periods as if that's a stat. Study the change. Just assume good faith.

Frankly whoever programmed `num_periods` had the mind of child at the time. And I'm sure they'd agree today.